packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -517,7 +518,12 @@ moduleFor(
             },
           })
         );
-        this.register('template:application_error', compile('Error! {{model.message}}'));
+        this.register(
+          'template:application_error',
+          compile(
+            EMBER_ROUTING_MODEL_ARG ? 'Error! {{@model.message}}' : 'Error! {{this.model.message}}'
```

Comment:
This change seems fine in the abstract, but we need to ensure that at least _some_ tests remain confirming all three scenarios work properly:

* implicit fallback
* `this.model`
* `@model`

Reply:
Addressed in the last commit